victorLi/assignments/treesDFS/lc112/lc112.js

```diff
@@ -0,0 +1,62 @@
+// Problem Statement #
+
+// Given a binary tree and a number ‘S’, find if the tree has a path from root-to-leaf such that the sum of all the node values of that path equals ‘S’.
```

Comment:
Make sure to include an overview of your approach, as well as a note about time and space complexity